docs/sources/developers/plugins/introduction-to-plugin-development/data-frames.md

```diff
@@ -1,5 +1,14 @@
 ---
 title: Data frames
+aliases:
+  - ../../plugins/data-frames/
+description: Learn about data frames and how they work in plugins.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - data frames
+  - dataframes
 ---
 
 # Data frames
```

Comment:
Both on local and latest docs https://grafana.com/docs/grafana/latest/developers/plugins/data-frames/ the ToC view is broken. 
<img width="291" alt="Screenshot 2023-06-27 at 17 53 22" src="https://github.com/grafana/grafana/assets/19860021/8fcf2ba4-d6bb-4b14-8c14-15684f2b913d">

I can fix it locally by changing `Data frame fields` and `Field Configurations` from H3 ### to H2 ##

Reply:
I have changed those two headers to H2 to solve the problem for now, but those headers were intended to be H3 and shouldn't have caused a TOC issue. I will investigate at a later point.